src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -400,58 +365,62 @@ const formConfig = {
-                        'view:privateMedicalRecordsReleasePermissionRestricted',
-                        'startTreatment', 'endTreatment',
-                        'treatmentCenterCountry', 'treatmentCenterStreet1',
-                        'treatmentCenterStreet2', 'treatmentCenterCity',
-                        'treatmentCenterState', 'treatmentCenterPostalCode'
+                        'country',
+                        'addressLine1',
+                        'addressLine2',
+                        'city',
+                        'state',
+                        'zipFirstFive'
                       ],
-                      treatmentCenterName: { // TODO: is this required?
-                        'ui:title': 'Name of private provider or hospital'
-                      },
-                      privateMedicalRecordsReleaseAccepted: {
-                        'ui:title': 'I give my consent, or permission, to my doctor to only release records related to this condition'
-                      },
-                      'view:privateMedicalRecordsReleasePermissionRestricted': {
-                        'ui:description': () => recordReleaseWarning,
-                        'ui:options': {
-                          expandUnder: 'privateMedicalRecordsReleaseAccepted'
-                        }
-                      },
-                      startTreatment: {
-                        'ui:widget': 'date',
-                        'ui:title': 'Approximate date of first treatment since you received your rating'
-                      },
-                      endTreatment: {
-                        'ui:widget': 'date',
-                        'ui:title': 'Approximate date of last treatment'
-                      },
-                      treatmentCenterCountry: { // TODO: need to update schema to use default def
+                      // TODO: confirm validation for PCIU address across all usage
+                      // 'ui:validations': [validateAddress],
+                      country: {
                         'ui:title': 'Country'
                       },
-                      treatmentCenterStreet1: {
-                        'ui:title': 'Street'
+                      addressLine1: {
+                        'ui:title': 'Street address'
                       },
-                      treatmentCenterStreet2: {
-                        'ui:title': 'Street'
+                      addressLine2: {
+                        'ui:title': 'Street address'
                       },
-                      treatmentCenterCity: {
+                      city: {
                         'ui:title': 'City'
                       },
-                      treatmentCenterState: {
+                      state: {
                         'ui:title': 'State'
                       },
-                      treatmentCenterPostalCode: {
+                      zipFirstFive: {
                         'ui:title': 'Postal code',
```

Comment:
We will probably want to specify ZIP here, unless @peggygannon indicates postal code is best, and perhaps obvious if we only display this field for addresses that specify 'USA' as the country.

Reply:
Just referenced the relevant page in the current mocks to get this display label. Happy to change if that's not what we want.